server/src/index.js

```diff
@@ -3,10 +3,14 @@ const bodyParser = require('body-parser');
 const db = require('../../database/index.js');
 const express = require('express');
 const app = express();
+const path = require('path');
 
 app.use(bodyParser.json());
 
-app.get('/', (req, res) => res.send('CavaTable reviews module server is working!'));
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + '../../../index.html'));
+    res.sendFile(path.join(__dirname + '../../../index.html'));
```

Comment:
Is there a reason for the duplicate lines here?

Reply:
Removed it! Nice catch.